src/realm/util/file.cpp

```diff
@@ -1564,7 +1569,19 @@ FileDesc File::get_descriptor() const
 bool File::get_unique_id(const std::string& path, File::UniqueID& uid)
 {
 #ifdef _WIN32 // Windows version
-    throw std::runtime_error("Not yet supported");
+    WindowsFileHandleHolder fileHandle(::CreateFile2(std::filesystem::path(path).c_str(), FILE_READ_ATTRIBUTES,
```

Comment:
Ugh, why would windows have you call a function named `CreateFile` if it don't want to actually create a file. Would you mind adding a comment like `// Despite the name, this call won't actually create a file if it doesn't exist yet.`